nilearn/datasets/utils.py

```diff
@@ -531,7 +538,10 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
         dt = time.time() - t0
         if verbose > 0:
             # Complete the reporting hook
-            sys.stderr.write(' ...done. (%i seconds, %i min)\n' % (dt, dt // 60))
+            sys.stderr.write(' ...done. ({0:.4f} MB, {1} seconds, '
+                             '{2} min)\n'
+                             .format(os.path.getsize(full_name) / 1e6,
+                                     int(dt), int(dt // 60)))
```

Comment:
Another slight niggle: you can use `'... {1:.0f} {2:.0f}'.format(dt, dt // 60) if you want to print without decimal digit. Also it will round to the closer int in contrast to your method.